### ISSUE TITLE
Visual Studio 2017 Attach to process very slow if 

### ISSUE THREAD

**Issue by Meister1977** [adamdriscoll/poshtools#627]
If I open the attach to process, the screen started flicking. Not just the VS itselfs, but if I use the Chrome in the foreground, it loses its focus multiple times.

Emma from MSFT found the root cause is PowerShell Tools. If I switched off, the Attach to process works again.
Original case:
https://developercommunity.visualstudio.com/content/problem/252462/attach-to-process-very-slow.html

**Comment by Mishu** [comment]
Yes, and the processes list is populated very very slow... The workaround right now for me is to uninstall the tools for now... Even disabling them behaves the same.
